Makefile

```diff
@@ -54,7 +48,6 @@ clean:
 	rm -f kprobemultipwru_bpf*
 	rm -f kprobepwruwithoutoutputskb_bpf*
 	rm -f kprobemultipwruwithoutoutputskb_bpf*
-	rm -rf ./release
 	cd libpcap/ && make clean
```

Comment:
There is no `Makefile` under `libpcap/` until we run `./configure` to generate one from `Makefile.in`, and `make clean` will fail with error ` No rule to make target 'clean' ` before that.